test/unit/solver/test_rosenbrock.cpp

```diff
@@ -134,25 +125,3 @@ TEST(RosenbrockSolver, DenseAlphaMinusJacobian)
       2);
 }
 
-TEST(RosenbrockSolver, Timing)
-{
-  auto solver = getSolver<micm::Matrix, SparseMatrix, micm::LinearSolver<double, SparseMatrix>>(1);
-
-  auto state = solver.GetState();
-
-  state.variables_[0] = {
-    1, 1, 1, 1, 1,
-  };
-
-  auto result = solver.Solve<false>(1, state);
-  EXPECT_EQ(result.stats_.total_forcing_time.count(), 0);
-  EXPECT_EQ(result.stats_.total_jacobian_time.count(), 0);
-  EXPECT_EQ(result.stats_.total_linear_factor_time.count(), 0);
-  EXPECT_EQ(result.stats_.total_linear_solve_time.count(), 0);
-
-  result = solver.Solve<true>(1, state);
-  EXPECT_NE(
-      result.stats_.total_forcing_time.count() + result.stats_.total_jacobian_time.count() +
-          result.stats_.total_linear_factor_time.count() + result.stats_.total_linear_solve_time.count(),
-      0.0);
-}
```

Comment:
Is there a reason that you delete this test?

Reply:
same as above.

Reply:
I added it back.